### ISSUE TITLE
[Occtax] - Contrôle des heures quand on est sur 2 mois distincts

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/GeoNature#1468]
Bonjour, 

Dans certains cas (31 aout -> 1er septembre par exemple), le controle de cohérence des heures n'est pas fonctionnel. 

Ainsi 31/08 20h -> 1/09 5h va renvoyer l'erreur " L'heure de début ne peut pas être supérieure à l'heure de fin "
Si en revanche je saisie "31/08 20h -> 2/09 5h" ça va fonctionner... C'est un peu un mystère pour le coup...

**Comment by camillemonchicourt** [comment]
Oui on s'est dit que c'était vraiment n'importe quoi de bosser dans la nuit du 31 août. 🤣

**Comment by DonovanMaillard** [comment]
Je suis d'accord. Mais que veux tu... on a des bénévoles qui forcent sur le Génépi et qui finissent plus tard que prévu :)
